aspnetcore/blazor/hybrid/class-libraries.md

```diff
@@ -26,6 +26,21 @@ The examples in this article share assets between a Blazor Server app and a .NET
 
 For guidance on creating an RCL, see <xref:blazor/components/class-libraries>. Optionally, access the additional guidance on RCLs that apply broadly to ASP.NET Core apps in <xref:razor-pages/ui-class>.
 
+:::moniker range="= aspnetcore-6.0"
+
+## Target framework
+
+To publish a WPF or Windows Forms project with a Razor class library (RCL) in ASP.NET Core 6.0, the RCL must have the [Target Framework Moniker (TFM)](/dotnet/standard/frameworks) set to `net6.0;net6.0-windows`.
```

Comment:
1. Is this true for all publishing types, or just ClickOnce?
2. What does it mean "in ASP.NET Core 6.0"?
3. Should it just say to _add_ `net6.0-windows`? Presumably there could already be some other various TFMs there, which might be fine?

cc @TanayParikh

Reply:
Made an update, but I rely upon @TanayParikh to indicate if they're sane changes. The current bit is now ...

> :::moniker range="= aspnetcore-6.0"
>
> \#\# Target framework
>
> To publish a WPF or Windows Forms project with a Razor class library (RCL) in .NET 6.0, the RCL must target \`net6.0-windows\`.
>
> Example:
>
> \`\`\`xml
> \<TargetFrameworks>net6.0;net6.0-windows</TargetFrameworks>
> \`\`\`
>
> For more information, see \[Target frameworks in SDK-style projects](/dotnet/standard/frameworks).
>
> :::moniker-end

Reply:
> Is this true for all publishing types, or just ClickOnce?

I believe this is for ClickOnce specifically, folder publishing should work without this change. 

> What does it mean "in ASP.NET Core 6.0"?
> To publish a WPF or Windows Forms project with a Razor class library (RCL) in .NET 6,

To publish a Blazor Hybrid WPF or Windows Forms project with a Razor class library (RCL) in .NET 6, the RCL...

> Should it just say to add net6.0-windows? Presumably there could already be some other various TFMs there, which might be fine?
> the RCL must target `net6.0-windows` in addition to `net6.0`.

Yep, I think that makes sense, thanks!

Reply:
Latest commit includes the ClickOnce piece with cross-links to ClickOnce guidance.